npm-shrinkwrap.json

```diff
@@ -1833,6 +2720,706 @@
       "from": "fs.realpath@>=1.0.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz"
     },
+    "fsevents": {
```

Comment:
It is of critical importance that we develop an emoji to express our feelings about shrinkwrap.